scrapers/LingerieTales.yml

```diff
@@ -0,0 +1,29 @@
+name: LingerieTales
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - lingerietales.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@class="title"]
+      Date:
+        selector: //time
+        postProcess:
+          - parseDate: January 2, 2006
+      Performers:
+        Name: //ul[@class='filters_buttons']/li[2]/a
+      Tags:
+        Name:
+          selector: //ul[@class="filters_buttons"]/li/a
+      Details:
+        selector: //div[@class="the_content_wrapper"]/p
```

Comment:
found a scene where the main desc is in a `pre` -> `https://www.lingerietales.com/video-item/leah-rising-early/`
```yaml
        selector: //div[@class="the_content_wrapper"]/p|//pre[@id="code"]
```
takes care of that

Reply:
Right behind ya hahahahaha, Bloody tags have screwed me over in this batch of studios DOH